webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -30,34 +31,39 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
       experimentId: trial.experimentId,
       trialId: trial.id,
     };
-  }, [ trial.bestAvailableCheckpoint, trial.experimentId, trial.id ]);
+  }, [ trial ]);
 
   const totalCheckpointsSize = useMemo(() => {
-    const totalBytes = trial.workloads
+    const totalBytes = trial?.workloads
       .filter(step => step.checkpoint
         && step.checkpoint.state === CheckpointState.Completed)
       .map(step => checkpointSize(step.checkpoint as CheckpointWorkload))
       .reduce((acc, cur) => acc + cur, 0);
+    if (!totalBytes) return;
     return humanReadableBytes(totalBytes);
-  }, [ trial.workloads ]);
+  }, [ trial?.workloads ]);
 
   const handleShowBestCheckpoint = useCallback(() => setShowBestCheckpoint(true), []);
   const handleHideBestCheckpoint = useCallback(() => setShowBestCheckpoint(false), []);
 
   return (
     <Section>
       <Grid gap={ShirtSize.medium} minItemWidth={180} mode={GridMode.AutoFill}>
-        {trial.runnerState && (
+        {trial?.runnerState && (
           <OverviewStats title="Last Runner State">
             {trial.runnerState}
           </OverviewStats>
         )}
-        <OverviewStats title="Start Time">
-          <TimeAgo datetime={trial.startTime} />
-        </OverviewStats>
-        <OverviewStats title="Total Checkpoint Size">
-          {totalCheckpointsSize}
-        </OverviewStats>
+        {trial?.startTime && (
+          <OverviewStats title="Start Time">
+            <TimeAgo datetime={trial.startTime} />
+          </OverviewStats>
+        )}
+        {totalCheckpointsSize && (
+          <OverviewStats title="Total Checkpoint Size">
+            {totalCheckpointsSize}
+          </OverviewStats>
+        )}
         {bestCheckpoint && (
```

Comment:
question: would something like `{bestCheckpoint && trial && (` make sense here? instead of `trial?.id` in title string?

Reply:
Yeah, that makes sense. I don't think there's any way for bestCheckpoint to exist without trial existing, but better to be explicit.